src/apis/objkt.ts

```diff
@@ -82,3 +117,21 @@ const buildGetHoldersInfoQuery = (address: string) => gql`
     }
   }
 `;
+
+const getCollectibleInfo = (address: string, tokenId: string) => gql`
```

Comment:
Please rename it to something like `buildGetCollectibleByAddressAndIdQuery`

Reply:
why? we don't need to save this info in the store